yt/yt/server/lib/chunk_pools/unittests/sorted_chunk_pool_new_keys_ut.cpp

```diff
@@ -109,6 +109,7 @@ class TSortedChunkPoolNewKeysTest
             MaxPrimaryDataWeightPerJob_,
             InputSliceDataWeight_,
             Inf64 /*inputSliceRowCount*/,
+            0 /*batchRowCount*/,
```

Comment:
This should be std::nullopt and instead of 0, forgot to change from the old implementation.